src/sInv.sol

```diff
@@ -31,35 +32,57 @@ interface IDistributor {
     function claimable(address) external view returns(uint);
```

Comment:
`IDistributor` interface is unused.

Reply:
Fixed in [c2beee4](https://github.com/InverseFinance/sINV/pull/2/commits/c2beee43668d7d25c5cf5a68f8199752667cc118)